transit/models/shipment.py

```diff
@@ -91,3 +91,17 @@ class ShipmentOrderMapping(BaseModel):
     class Meta:
         managed = True
         db_table = 'ShipmentOrderMapping'
+
+
+# Impossible to put this function inside of class due to lambda serialization exception
+def content_file_name(instance, filename):
+    return '/'.join(['shipment', str(instance.shipment.id), filename])
```

Comment:
'/' -> what about another OS like windows? But I assume we will run only on linux, right?

Reply:
That's correct we're focusing on unix, but either way Windows should resolve slashes just fine.

Reply:
ok